ajax_select/__init__.py

```diff
@@ -13,6 +13,9 @@
 from django.utils.encoding import force_text
 from django.utils.html import escape
 from django.utils.translation import ugettext_lazy as _
+from django.utils.module_loading import autodiscover_modules
```

Comment:
Ah yes, I completely forgot to bring this up.  I did not add version checking, but I can definitely do this, if you'd like to continue support for 1.5/1.6.  I'll make some changes for this.

Autodiscover is most likely to be used if you are making a large number of custom lookup channels across various apps in your project that subclass LookupChannel.  I should probably add documentation that you must use "lookups.py" as the module name in order for autodiscover() to work.  You register the custom lookups with @ajax_select.register('label'), and the autodiscover function is able to "run" those decorators upon initialization of the app, when it forces an import on all of those" lookups.py" files.  This collects the labels into the registry without having to go into settings.py.  This is similar to how the django admin autodiscovers ModelAdmin classes, except we are storing a registry of dict-like labels for channels, rather than admin class instances.  I'm actually fairly new to the source for autodiscover_modules() myself, so I hope I understand it correctly.  Does that answer your question?